docs/api/endorsement-provisioning/endorsement-provisioning.yaml

```diff
@@ -0,0 +1,76 @@
+openapi: '3.0.0'
+
+info:
+  title: attestation endorsement provisioning
+  description: For further details, see datatracker.ietf.org/doc/draft-birkholz-corim
+  version: '1.0.0alpha'
+servers:
+  - url: 'https://veraison.example/endorsement-provisioning/v1'
+paths:
+  /submit:
+    post:
+      description: >
+        Submit a single CoRIM or a batch of CoRIM payloads using this API.
+      requestBody:
+        required: true
+        content:
+          application/rim+cbor:
+            schema:
+              $ref: './schemas/components.yaml#/components/schemas/CoRIMData'  
+      responses:
+        '200':
+          description: >
+            A new CoRIM or a batch of CoRIM has been successfully provisioned inside 
+            a Verifier. The transaction is complete.
+          content:
+            application/json:
+              schema:
+                $ref: './schemas/components.yaml#/components/schemas/CoRIMStatus'
+        '202':
+          description: >
+            The submission is successful and will be processed
+            asynchronously. A new session resource is created and its URL is returned in the
+            Location header.  The resource will be kept alive for the specified
+            time-to-live.  The new resource comprises the handle to be used to check the status
+            of the ongoing request.
+          headers:
+            Location:
+              schema:
+                type: string
+                format: uri
+              description:
+                URL of the newly created session resource to be used in
+                subsequent status check.  (The current content of the
+                resource is returned in the response body.)
+          content:
+            application/json:
+              schema:
+                $ref: './schemas/components.yaml#/components/schemas/CoRIMResponse'
+        default:
+          $ref: './schemas/components.yaml#/components/responses/Error'
+  /session/{handle}:
+    get:
+      description: >
+        retrieve the endorsement-provisioning session resource
+      parameters:
+        - $ref: '#/components/parameters/handle'
+      responses:
+        '200':
+          description: >
+            The handle or identifier which was returned at the time of CoRIM payload submission
+          content:
+            application/json:
+              schema:
+                $ref: './schemas/components.yaml#/components/schemas/CoRIMStatus'
```

Comment:
SessionResource will be returned here: So we are now returnfing the resource, as returned in POST 201 response!